etc/mpv-powershell-completion.ps1

```diff
@@ -0,0 +1,285 @@
+# PowerShell command line completion for the mpv media player.
+
+# It can be installed by dot sourcing it in the PowerShell profile.
+
+#
+# This file is part of mpv.
+#
+# mpv is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# mpv is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with mpv.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+$Options = New-Object Collections.Generic.List[Object]
+$UpdateOptions = @('vo', 'ao', 'profile', 'hwdec', 'audio-device', 'scale', 'tscale',
+                   'error-diffusion', 'vulkan-device')
```

Comment:
More options also support `help`.

Reply:
Single value options or list (or key value) options, that might be difficult to build and rarely be used. Does the bash script support all options?

Reply:
Not sure.

`--cscale --dscale --d3d11-adapter --vaapi-device` and more should be included with no problem.

Reply:
It can be added, but would need some n00b friendly git advice in order to not screw up again. Preferable using GitHub Desktop or VS Code, command line is also okay.

Reply:
I tried --vaapi-device=help on wsl and real ubuntu, it did not work here.

Reply:
I've added the missing options, and some code quality improvements. It's done as for as I can tell.

Reply:
```
--audio-channels
--drm-connector --drm-mode
--sws-scaler
--scale-window --tscale-window --cscale-window --dscale-window
--vulkan-display-display --vulkan-display-mode --vulkan-display-plane
```

Reply:
For reference, most of these options aren't completed in zsh either. I think they should all be converted to `OPT_CHOICE_C` so that they will also be completed in the console and completion scripts will not have to hardcode specific options.

Reply:
Some Vulkan options did not work, if I remember correctly. I did not add options that I don't understand, or would be too difficult to implement, or wouldn't be very useful.

Reply:
> The index of the display, on the selected Vulkan device, to present on when using the displayvk GPU context. 

I guess your build didn't have displayvk backend, so those options didn't work for you.